content/applications/inventory_and_mrp/quality/quality_reporting/quality_checks_report.rst

```diff
@@ -0,0 +1,195 @@
+=====================
+Quality checks report
+=====================
+
+In Odoo's *Quality* app, the *Quality Checks* report displays key statistics about existing quality
+checks. This includes the number of quality checks in each stage, the dates on which checks were
+passed or failed, the products for which checks were created, and more.
+
+The report offers a number of options to help visualize the data, including customizable views and
+three different graph types. This allows users to extract deeper insights from their quality check
+processes.
+
+Report dashboard
+================
+
+To access the :guilabel:`Quality Checks` report dashboard, navigate to :menuselection:`Quality app
+--> Reporting --> Quality Checks`.
+
+There are four main views that can be used to display quality check data:
+
+- :ref:`Graph view <quality_alerts_report/graph-view>`
+- :ref:`Pivot view <quality_alerts_report/pivot-view>`
+- :ref:`Kanban view <quality_alerts_report/kanban-view>`
+- :ref:`List view <quality_alerts_report/list-view>`
+
+By default, the dashboard displays data in the graph view, using the bar chart graph type. To select
+a different view, click on the corresponding button in the top-right corner of the dashboard.
+
+.. figure:: quality_checks_report/view-select.png
+   :align: center
+   :alt: The view select buttons on the quality checks report dashboard.
+
+   The view select buttons on the quality checks report dashboard. From left to right: Graph, Pivot,
+   Kanban, and List.
+
+.. _quality_alerts_report/graph-view:
+
+Graph view
+----------
+
+To display quality check data in graph view, click the :icon:`fa-area-chart` :guilabel:`(Graph)`
+button in the top-right corner of the dashboard.
+
+There are three chart types that can be used to visualize quality check data:
+
+- :ref:`Bar chart <quality_alerts_report/bar-chart>`
+- :ref:`Line chart <quality_alerts_report/line-chart>`
+- :ref:`Pie chart <quality_alerts_report/pie-chart>`
+
+By default, graph view displays data using the bar chart graph type. To select a different graph
+type, click the corresponding button in the top-left corner of the dashboard.
+
+.. figure:: quality_checks_report/graph-select.png
+   :align: center
+   :alt: The graph select buttons on the quality checks report dashboard.
+
+   The graph select buttons on the quality checks report dashboard. From left to right: Bar Chart,
+   Line Chart, and Pie Chart.
+
+.. _quality_alerts_report/bar-chart:
+
+Bar chart
+~~~~~~~~~
+
+To display quality check data in a bar chart, click the :icon:`fa-bar-chart` :guilabel:`(Bar Chart)`
+button in the top-left corner of the dashboard.
+
+The x-axis of the bar chart shows the three stages that a quality check can be organized under:
+:guilabel:`Failed`, :guilabel:`To do`, and :guilabel:`Passed`. The y-axis of the bar chart shows the
+number of quality checks in each stage, which is displayed by a bar above each stage name.
```

Comment:
With this just being the default setting, and other options available via the Measures drop-down, and various filter/grouping options in the search bar mega menu, do you think it might be best to mention that this is the default, and those other options exist --  that way the reader doesn't feel like this is all there is... just a thought